jgrapht-core/src/main/java/org/jgrapht/alg/VertexCovers.java

```diff
@@ -57,6 +57,7 @@
  * @author Linda Buisman
  * @since Nov 6, 2003
  */
+@Deprecated
```

Comment:
When adding the @Deprecated annotation, it's good to add a companion Javadoc @deprecated which directs developers to the replacement.